### ISSUE TITLE
laravel 5.4 - Undefined property: Illuminate\Support\MessageBag::$default

### ISSUE THREAD

**Issue by iateadonut** [dwightwatson/bootstrap-form#86]
In Laravel 5.4,

If you create a input field like this:
`{!! BootForm::text('address') !!}`

And in your controller, validate:
```
        $rules = [
            'address'=>'required',
        ];
        $validator = Validator::make($request->input(), $rules);

        if ( $validator->fails() ) {
            return redirect()->route('post-company-create')
                ->with(['errors'=>$validator->errors()]);
        }
```

You'll receive this error if you submit a blank 'address':
`
Undefined property: Illuminate\Support\MessageBag::$default`

**Comment by dwightwatson** [comment]
Did you just start a new project with this and run into the error, or was it something that happened after upgrading your existing project? I'm wondering if perhaps this has snuck in a recent Laravel update and we need to update it on our end.

**Comment by iateadonut** [comment]
New project, laravel 5.4.19.

**Comment by arnevanhoorn** [comment]
I have the same issue, any updates?

**Comment by dwightwatson** [comment]
I was unable to replicate this issue, so no progress has been made. If anyone is able to provide more information I'll be more than happy to look into it further.

**Comment by vedmant** [comment]
I have the same issue, added a pull request with simple fix for this issue https://github.com/dwightwatson/bootstrap-form/pull/94, please release it if possible

**Comment by dwightwatson** [comment]
I still don't understand why the issue is occurring for some projects, but I've created a PR #95 which uses the workaround that worked in #94 but makes it configurable option. If anyone is able to pull that branch down, have a play with it and see if it solves the issue for them that would be great. If all is well I'll merge and tag the release.

**Comment by vedmant** [comment]
The problem is that ` $this->form->getSessionStore()->get('errors')` already returns a message bag. So the option what would work should be `null` and then the code that gets field error is:

    $errorBag = $this->getErrorBag() ? $this->getErrors()->{$this->getErrorBag()} : $this->getErrors();

**Comment by dwightwatson** [comment]
Should be resolved in `1.1.18`
